package.json

```diff
@@ -145,7 +145,7 @@
     "clean": "rimraf ./coverage ./es5 ./dist"
   },
   "dependencies": {
-    "@twilio/webrtc": "4.4.0",
+    "@twilio/webrtc": "github:twilio/twilio-webrtc.js#49f022079ec4d57bf41b71a414845c2ef0dcb121",
```

Comment:
Should this be `5.0.0` by chance?

Reply:
@PikaJoyce It will be eventually when we release 5.0.0.